packages/gatsby-plugin-sharp/src/plugin-options.js

```diff
@@ -11,6 +11,9 @@ const pluginDefaults = {
 
 const generalArgs = {
   quality: 50,
+  jpeg_quality: ``,
+  png_quality: ``,
+  webp_quality: ``,
```

Comment:
Having those args as empty strings seem a bit weird. Maybe we could change them to `null` (or `undefined`) and make sure they are handled properly in the `healOptions` function (with `options.quality` as a fallback as suggested by @coreyward below).